static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -326,7 +326,9 @@ define([
                         resolve();
                     }
 
-                    if (abVideoAutoplay.variant === 'autoplay') {
+                    var contentType = config.page.contentType;
+                    var isArticleOrLiveBlog = contentType === 'Article' || contentType === 'LiveBlog';
+                    if (abVideoAutoplay.variant === 'autoplay' && isArticleOrLiveBlog) {
```

Comment:
isInVariant it just using the same logic it uses to run the code in your module.  Often the code in the module is empty, but if the test is off/expired but you are in the bucket, you can end up with some weird situation where you're half in the test.  Also the code for updating your local storage relies on the switch etc.  So I think in reality the code needs updating/refactoring to support test mode where you are in the test on the client side but it doesn't send back analytics